jira_metrics/cycle_time.py

```diff
@@ -22,9 +22,65 @@
 SECONDS_TO_HOURS = 3600
 
 
+def convert_raw_issue_to_simple_object(raw_issue):
```

Comment:
We really should move this to jira_utils instead and us it to return the tickets in the same format as we are used to in the other files